packages/mdc-base/component.ts

```diff
@@ -118,4 +118,6 @@ export class MDCComponent<FoundationType extends MDCFoundation> {
   }
 }
 
+// tslint:disable:no-default-export Needed for backward compatibility with
+// MDC Web v0.44.0 and earlier.
```

Comment:
we don't need this in the other components/foundations/adapters? Does this pass if we revert to:

```
export {MDCComponent, MDCComponent as default}
```

Reply:
It still complains with `export {MDCComponent, MDCComponent as default}`.

In `master`, the only files that have default exports are all the foundations,
`mdc-auto-init/index.js`, and `mdc-base/component.js`:

https://github.com/material-components/material-components-web/blob/70672f12ff19591c53fef6876c059c854f9cb816/packages/mdc-base/component.js#L131